config/docker-compose.yaml

```diff
@@ -44,9 +55,73 @@ services:
       - "--match"
       - "*"
 
+  # TODO move variables out
```

Comment:
Where? What do you mean?

Reply:
I've already done that, I forgot to remove the comment. Thanks!